docs/cn/Configuring-Alluxio-with-GCS.md

```diff
@@ -22,7 +22,9 @@ priority: 0
 
 # 配置Alluxio
 
+接着，如果还没有配置，请用‘bootstrap-conf'命令创建自己的配置。
```

Comment:
Can you double check the English version? this should be updated in the "Initial Setup" section.
Also the content seems not matching with the eng doc...